app/views/feed/contentSelectorPreview.phtml

```diff
@@ -1,30 +1,36 @@
 <?php FreshRSS::preLayout(); ?>
 <!DOCTYPE html>
-<html class="preview_background">
+<html class="preview_background" lang="<?= FreshRSS_Context::$user_conf->language ?>" xml:lang="<?= FreshRSS_Context::$user_conf->language ?>">
 	<head>
 		<?= self::headStyle() ?>
-		<script src="<?= Minz_Url::display('/scripts/preview.js?' . @filemtime(PUBLIC_PATH . '/scripts/main.js')) ?>"></script>
+		<script src="<?= Minz_Url::display('/scripts/preview.js?' . @filemtime(PUBLIC_PATH . '/scripts/preview.js')) ?>"></script>
```

Comment:
Oh ! It's why my script had difficulties to be re-downloaded by my browser :/

Reply:
That would be a good reason ^^ We could imagine a system that would calculate the `filemtime` automatically to avoid such problems (e.g. by introducing a `Minz_Url::static('/scripts/preview.js')` method), but it's very low in my priorities